src/jti_background.js

```diff
@@ -178,6 +206,7 @@ function downloadJSONData (url, callback) {
         if (status) {
             data.templates = JSONtoTemplateData(data.templates);
             data.options.domains = JSONtoDomainData(data.options.domains);
+            data.options.inputIDs = JSONtoInputIDData(data.options.inputIDs);
```

Comment:
These three lines of code are repeated numerous times.
Right here,
https://github.com/rdbrck/jira-template-injector/pull/44/files#diff-9566d9c6e04bce6d87d5cbfbf1062b04R196,
https://github.com/rdbrck/jira-template-injector/pull/44/files#diff-9566d9c6e04bce6d87d5cbfbf1062b04R131,
https://github.com/rdbrck/jira-template-injector/pull/44/files#diff-9566d9c6e04bce6d87d5cbfbf1062b04R438

I think we should factor this out into its own function.

Reply:
This refactoring has been done in branch EXT-1000-JSON-parsing.
<img width="569" alt="screen shot 2018-01-30 at 2 39 47 pm" src="https://user-images.githubusercontent.com/28017034/35595411-7d535ef2-05cb-11e8-9fb0-c3abde002a75.png">